src/cbmc_starter_kit/template-for-repository/proofs/Makefile.common

```diff
@@ -272,6 +336,22 @@ CBMC_OBJECT_BITS ?= 8
 # of times CBMC actually unwinds the loop.
 UNWINDSET ?=
 
+# CBMC early loop unwinding (Normally set in the proof Makefile)
+#
+# Most users can ignore this variable.
+#
+# This variable exists to support the use of loop and function
+# contracts, two features under development for CBMC.  Checking the
+# assigns clause for function contracts and loop invariants currently
+# assumes loop-free bodies for loops and functions with contracts
+# (possibly after replacing nested loops with their own loop
+# contracts).  To satisfy this requirement, it may be necessary to
+# unwind some loops before the function contract and loop invariant
+# transformations are applied to the goto program.  This variable
+# EARLY_UNWINDSET is identical to UNWINDSET, and we assume that the
+# loops mentioned in EARLY_UNWINDSET and UNWINDSET are disjoint.
+EARLY_UNWINDSET ?=
```

Comment:
Will we need to change `UNWINDSET` to `EARLY_UNWINDSET` in the function contract work?